docs/releases/2020s2r1.md

```diff
@@ -39,8 +39,8 @@ These are the user stories that where included in 2020s2r1:
 | [Web-API#78](https://github.com/aau-giraf/web-api/issues/78)                                | Group sw607f20         |
 | [Web-API#94](https://github.com/aau-giraf/web-api/issues/94)                                | Group sw616f20         |
 | [Web-API#96](https://github.com/aau-giraf/web-api/issues/96)                                | Group sw616f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw607f20         |
-| [API_client#78](https://github.com/aau-giraf/api_client/issues/78)                          | Group sw610f20         |
+| [API_client#24](https://github.com/aau-giraf/api_client/issues/24)                          | Group sw607f20         |
+| [API_client#33](https://github.com/aau-giraf/api_client/issues/33)                          | Group sw610f20         |
```

Comment:
From what I can tell, @Laexander created this issue and he is in group 15
```suggestion
| [API_client#33](https://github.com/aau-giraf/api_client/issues/33)                          | Group sw615f20         |
```

Reply:
Yes group 15 made the issue, but group 7 fixed it. It should be correct now:
https://github.com/aau-giraf/api_client/issues/33
https://github.com/aau-giraf/api_client/issues/24